### DISCUSSION TITLE
Regarding : Required the sample dimension tables and data files for Teacher / Student attendance for ingestion

### DISCUSSION THREAD

**Discussion by vskmizoram** [Sunbird-cQube/community#862] [Q&A]
Need, the sample dimension and data files for the ingestion of teacher attendance and Student attendance data

**Comment by bommalla** [comment]
@vskmizoram 

Thanks for reaching us. I'll send you the templates

**Comment by bommalla** [comment]
@vskmizoram 

As per your request, please find below the VSK cqube template. This template contains all six starter pack programs, new programs, along with dimensions. Based on this template, we need to prepare the data and proceed with the ingestion process.

https://docs.google.com/spreadsheets/d/10yWmivrqtxdRLiJpX_ytzxbvoGG0WNWEI-7RLV8zelM/edit?pli=1#gid=208908870

**Reply by vskmizoram** [reply]
Thanks for providing the format for dimensions and data files.

The data have been ingested successfully.

Now I need to modify the data for the single application after the visualization . Kindly send me the procedure for updation.

**Comment by bommalla** [comment]
@vskmizoram 

Thanks for the update, Please find the below steps to run single application.

Step1: First go inside nifi_app container by using below command 
sudo docker exec -it nifi_app bash
Step2: Then navigate the below mentioned path
cd Sunbird-cQube-processing-ms/impl/c-qube/ingest
Step3: After ingested the data then you can run below command
yarn cli ingest-data --filter='pgi'

**Reply by bommalla** [reply]
@vskmizoram 

As per your request please find the reference dimensions from end  

[reference_dimensions.zip](https://github.com/Sunbird-cQube/community/files/14358800/reference_dimensions.zip)

**Comment by vskmizoram** [comment]
Thanx for your prompt support..

we need to activate the teacher attendance option on the dashboard. kindly guide us how to do that..

**Reply by bommalla** [reply]
@vskmizoram 

Thanks for reaching out,regarding this issue please go inside query-builder container by using below command.

sudo docker exec -it querybuilder_app sh

There is one program_selector.csv file is there.Under program_selector.csv file check if teacher_attendance is there or not ?  if it's not there please add it and restart the querybuilder.

**Comment by vskmizoram** [comment]
Tx for your prompt support...

one small issue coming... the login dashboard not working

![image](https://github.com/Sunbird-cQube/community/assets/157444637/38119a49-cf12-445c-97b6-816f4ab701c3)

kindly provide the solution

**Reply by bommalla** [reply]
@vskmizoram 

Thanks for reaching out. Please go inside query builder  by using below command

sudo docker exec -it querybukder_app sh
apk add vim
vim .env
please check keycloak url and changed to http://keycloak_app:8080/auth

**Reply by vskmizoram** [reply]
I need help on .. why PGI data is not coming at the dashboard.. even while clicking the details.. the summary data is also not coming...

**Comment by bommalla** [comment]
@vskmizoram 

Thanks for reaching out, Can you please tell which version of cQube are you installing ?
